hoomd/jit/CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ pybind11_add_module(_${PACKAGE_NAME} SHARED ${_${PACKAGE_NAME}_sources} ${_${PAC
 add_library(HOOMD::_${PACKAGE_NAME} ALIAS _${PACKAGE_NAME})
 
 add_library (_${PACKAGE_NAME}_llvm SHARED ${_${PACKAGE_NAME}_llvm_sources})
+set_property(TARGET _${PACKAGE_NAME}_llvm PROPERTY CXX_STANDARD 14)
```

Comment:
This is no longer necessary. We now set a project wide `CMAKE_CXX_STANDARD 14`.

Reply:
eliminated